Dockerfile

```diff
@@ -18,7 +18,7 @@ RUN apt-get update -y && \
 RUN apt-get update -y && \
   apt-get upgrade -y && \
   apt-mark hold php-apcu-bc && \
-  apt-get install -y apache2 libapache2-mod-php5.6 libxml2-utils git-core unzip npm nodejs-legacy wget fontconfig libaio1 php5.6 php5.6-dev php5.6-xml php5.6-mbstring php5.6-curl php5.6-gd php5.6-zip php5.6-intl php5.6-sqlite3 php5.6-mysql php5.6-pgsql php5.6-soap php5.6-phpdbg php-redis php-memcached php-imagick php-smbclient php-apcu && \
+  apt-get install -y apache2 libapache2-mod-php5.6 libxml2-utils git-core unzip npm nodejs-legacy wget fontconfig libaio1 php5.6 php5.6-dev php5.6-xml php5.6-mbstring php5.6-curl php5.6-gd php5.6-zip php5.6-intl php5.6-sqlite3 php5.6-mysql php5.6-pgsql php5.6-soap php5.6-phpdbg php-redis php-memcached php-imagick php-smbclient php-apcu php5.6-ldap && \
```

Comment:
@individual-it 
I think you missed updating your branch before adding php-ldap - the Dockerfile in 5.6 branch has yarn and nodejs from the vendor repository

Reply:
🤦‍♂️ should be fixed now